pkg/asset/installconfig/platformpermscheck.go

```diff
@@ -90,7 +91,14 @@ func (a *PlatformPermsCheck) Generate(dependencies asset.Parents) error {
 	case ibmcloud.Name:
 		// TODO: IBM[#90]: platformpermscheck
 	case powervs.Name:
-		//@TODO add check that the account plan is anything but "lite"
+		bxCli, err := powervsconfig.NewBxClient()
```

Comment:
we should work with the ibm x86 team to see if this can be common for both.

Reply:
@bkhadars did you already take a look at the ibmcloud code? Is it possible for us to share theirs? We may have to tweak where we store a credential, etc. But for 4.11 if that's too much maybe we don't have time to tweak that.

Reply:
@clnperez currenlty ibmcloud is not doing the permission check https://github.com/openshift/installer/blob/950155cddc4b8018b9c6ea4de84695c558b77d24/pkg/asset/installconfig/platformpermscheck.go#L90, I have looked into the ibmcloud code as well and we can't use their code for permissioncheck validation as of now